docs/resiliency/disaster-recovery-azure-applications.md

```diff
@@ -36,31 +36,31 @@ For more information on how Azure Traffic Manager works, refer to:
 * [Configure failover routing method](/azure/traffic-manager/traffic-manager-configure-failover-routing-method/)
 
 ## Azure disaster scenarios
-The following sections cover several different types of disaster scenarios. Region-wide service disruptions are not the only cause of application-wide failures. Poor design or administration errors can also lead to outages. It's important to consider the possible causes of a failure during both the design and testing phases of your recovery plan. A good plan takes advantage of Azure features and augments them with application-specific strategies. The chosen response is dictated by the importance of the application, the recovery point objective (RPO), and the recovery time objective (RTO).
+The following sections cover several different types of disaster scenarios. Region-wide service disruptions are not the only cause of application-wide failures. Poor design and administrative errors can also lead to outages. It's important to consider the possible causes of a failure during both the design and testing phases of your recovery plan. A good plan takes advantage of Azure features and augments them with application-specific strategies. The chosen response is determined by the importance of the application, the recovery point objective (RPO), and the recovery time objective (RTO).
 
 ### Application failure
-Azure Traffic Manager automatically handles failures that result from the underlying hardware or operating system software in the host virtual machine. Azure creates a new instance of the role on a functioning server and adds it to the load-balancer rotation. If the number of role instances is greater than one, Azure shifts processing to the other running role instances while replacing the failed node.
+Azure Traffic Manager automatically handles failures that result from the underlying hardware or operating system software in the host virtual machine. Azure creates a new role instance and adds it to theavailable pool. If the more than one role instance was already running, Azure shifts processing to the other running role instances while replacing the failed node.
```

Comment:
- theavailable  - missing a space
- If the more than one role - extra word here